### DISCUSSION TITLE
Rotation matrix

### DISCUSSION THREAD

**Discussion by prasanta13** [charnley/rmsd#82] [General]
Great effort. Can this tool utilized to get only the rotation from one structure to another? Suppose I want to check the rotation of ethane in one .xyz file to another .xyz file?

**Comment by charnley** [comment]
Hi, do you mean from the command line? Because `rmsd.kabsch(P, Q)` calculates the rotation matrix between vector/coordinates/structure P and Q.

**Reply by prasantb-iitk** [reply]
Exactly, I was trying to find the rotation matrix between P and Q points using my own function in python3 with jupyter notebook, but I am not getting desired output.. Lets try with your code..
